File: Pertemuan1.3-CPP/Titik.cpp
class Titik
{
    /* Kelas yang digunakan untuk mengimplementasikan sebuah tipe titik */

private:
    int x; /*koordinat x*/
    int y; /* koordinat y */

public:
    Titik()
    {
        /* konstruktor */
        x = 0;
        y = 0;
    }

    Titik(int xp, int yp)
    {
        /* konstruktor */
        x = xp;
        y = yp;
    }

    void setX(int xp)
    {
        /* mengeset nilai koordinat x */
        x = xp;
    }

    int getX()
    {
        /* mengembalikan nilai koordinat x */
        return x;
    }

    void setY(int yp)
    {
        /* mengeset nilai koordinat x */
        y = yp;
    }

    int getY()
    {
        /* mengembalikan nilai koordinat x */
        return y;
    }

    ~Titik()
    {
        /* destruktor */
    }
};

File: Pertemuan1.2-Python/Titik.py
class Titik:
    """ 
    kelas yang digunakan untuk mengimplementasikan  
    sebuah tipe titik
    """

    def __init__(self, x, y):
        # konstruktor
        self.x = x
        self.y = y

    def setX(self, x):
        # mengeset nilai koordinat x
        self.x = x

    def getX(self):
        # mengembalikan nilai koordinat x
        return self.x

    def setY(self, y):
        # mengeset nilai koordinat y
        self.y = y

    def getY(self):
        # mengembalikan nilai koordinat y
        return self.y
